fronts-client/src/constants/feastPalettes.ts

```diff
@@ -141,6 +141,52 @@ export const feastCollectionPalettes: PaletteOption[] = [
 		},
 		imageURL: 'https://uploads.guim.co.uk/2024/03/21/Baking.png',
 	},
+	{
+		id: 'berries',
+		name: 'Berries',
+		palettes: {
+			light: {
+				foregroundHex: '#BB3C81',
+				backgroundHex: '#F9F9F5',
+			},
+			dark: {
+				foregroundHex: '#FCF0F6',
+				backgroundHex: '#363632',
+			},
+		},
+		imageURL: ' https://uploads.guim.co.uk/2024/11/18/Berries.png',
+	},
+	{
+		id: 'blackeberries_blueberries',
+		name: 'Blackeberries\nBlueberries',
```

Comment:
There’s a spelling mistake here: the correct spelling would be `blackberries_blueberries`, without the extra e. Can we change it, or does it need to be this way to match an existing name somewhere else?

Reply:
Ah nice spot! Yes, easy to change. I will remove extra `e` from `Blackeberries` 😀

Reply:
Thanks @emdash-ie, plz recheck when you can